perf_test/blas/blas2/KokkosBlas_dot_perf_test_rps.cpp.reference

```diff
@@ -0,0 +1,99 @@
+/*
+//@HEADER
```

Comment:
This is a reference file?

Reply:
@srajama1 - yes, it is.  I'll remove it, as it's not pertinent to the gemv test.